docs/source/getting_started/installation.rst

```diff
@@ -114,4 +114,78 @@ A tool like `postcss <https://postcss.org/>`__ can be used to convert the CSS fi
 ``jupyterlab/build`` directory manually if desired.
 
 
+Installation behind firewall
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your computer is behind corporate proxy or firewall,
+you may encounter an HTTP and SSL errors due to custom security profiles managed by corporate IT departments.
+
+This is expected, because your company can block connection to widely used repositories in Python community.
+
+Good start is to ask your network administrator, to allow all http + https communication to widely used Python / Jupyter servers:
+
+- \*.pypi.org
+- \*.pythonhosted.org
+- \*.continuum.io
+- \*.anaconda.com
+- \*.conda.io
+- \*.github.com
+- \*.githubusercontent.com
+- \*.npmjs.com
+- \*.yarnpkg.com
+
+Alternatively you can specify proxy user (mostly domain user with password),
+that is allowed to communicate via network. This can be easily achieved
+by setting 2 common environment variables `HTTP_PROXY` and `HTTPS_PROXY`.
+These variables are automatically used by many open-source tools (like ``conda``), if setup correctly.
+
+.. code:: bash
+
+    # For Windows
+    set HTTP_PROXY=http://USER:PWD@proxy.company.com:PORT
+    set HTTPS_PROXY=https://USER:PWD@proxy.comp any.com:PORT
+
+    # For Linux / MacOS
+    export HTTP_PROXY=http://USER:PWD@proxy.company.com:PORT
+    export HTTPS_PROXY=https://USER:PWD@proxy.company.com:PORT
+
+
+In case you can communicate via HTTP, but installation with ``conda`` fails
+on connectivity problems to HTTPS servers, you can disable using SSL for ``conda``.
+
+.. warning:: Disable SSL in communication is generally not recommended and involves potential security risk.
+    But for testing purposes, this step can help you to get rid of all HTTPS problems in ``conda`` quickly.
```

Comment:
```suggestion
.. warning:: Disable SSL in communication is generally not recommended and involves potential security risk.
```

Reply:
It seems this was not resolved? The purpose here was to not undercut the effectiveness of the warning.

Reply:
jj, clear now ;)
(for both occurences)